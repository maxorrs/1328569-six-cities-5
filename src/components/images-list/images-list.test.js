import React from 'react';
import renderer from 'react-test-renderer';
import ImagesList from './images-list';

describe(`ImagesList is rendered correctly`, () => {
  it(`ImagesList is rendered correctly`, () => {
    const tree = renderer
      .create(
          <ImagesList
            images={[`1`, `2`]}/>
      )
    .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
