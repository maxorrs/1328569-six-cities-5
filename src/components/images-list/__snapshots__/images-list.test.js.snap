// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImagesList is rendered correctly 1`] = `
<div
  className="property__gallery-container container"
>
  <div
    className="property__gallery"
  >
    <div
      className="property__image-wrapper"
    >
      <img
        alt="Photo studio"
        className="property__image"
        src="1"
      />
    </div>
    <div
      className="property__image-wrapper"
    >
      <img
        alt="Photo studio"
        className="property__image"
        src="2"
      />
    </div>
  </div>
</div>
`;
